docs/spark/tutorials/get-started.md

```diff
@@ -142,43 +144,49 @@ The `dotnet` command creates a `new` application of type `console` for you. The
 
 To use .NET for Apache Spark in an app, install the Microsoft.Spark package. In your command prompt or terminal, run the following command:
 
-`dotnet add package Microsoft.Spark --version 0.8.0`
+`dotnet add package Microsoft.Spark`
+
+> [!NOTE]
+> This tutorial uses the latest version of the `Microsoft.Spark` NuGet package unless otherwise specified.
 
 ### 3. Code your app
 
 Open *Program.cs* in Visual Studio Code, or any text editor, and replace all of the code with the following:
 
 ```csharp
 using Microsoft.Spark.Sql;
+using static Microsoft.Spark.Sql.Functions;
 
-namespace MySparkApp
+namespace mySparkApp
```

Comment:
I believe the convention for naming namespaces is PascalCase. Why this is changed to camelCase?